pkg/repository/aws/dynamo.go

```diff
@@ -73,9 +73,8 @@ func (s *Dynamo) GetPrimaryKey(pk string, sk string) PrimaryKey {
 	}
 }
 
-func (s *Dynamo) GetDBItem(tableName string, pk string, sk string) (map[string]*dynamodb.AttributeValue, error) {
-	// fmt.Println("Calling GetDBItem", tableName, pk, sk)
-
+func (s *Dynamo) GetDBItem(tableName string, pk string, sk string) (*models.CrowdactionData, error) {
```

Comment:
If we return `*models.CrowdactionData` I think the method name should reflect that, since we will need to return other objects from DynamoDB as well.

Reply:
You have an excellent point, this was done in one of the commits but then modified in the final version that was committed for review.